File: acm_statistic_backend/api/api/DTO/evolutionTemporel/ComparaisonDynamiqueDTO.cs
namespace api.DTO.evolutionTemporel
{
    public class ComparaisonDynamiqueDTO
    {
        public int Mois { get; set; }

        public String? Mois_Lettre { get; set; }
        public decimal Total_Annee { get; set; }
        public decimal Total_Annee_Precedente { get; set; }
        public decimal Difference { get; set; }
        public decimal? Pourcentage_Variation { get; set; }

        public decimal? Variation_mois { get; set; }
    }
}

File: acm_statistic_frontend/Interface Tableau de Bord Analytique Aérien/src/models/evolutionTemporel/ComparaisonDynamiqueDTO.ts
export class ComparaisonDynamiqueDTO{
    mois: number;
    mois_Lettre: string | null;
    total_Annee: number;
    total_Annee_Precedente: number;
    difference: number;
    pourcentage_Variation: number | null;
    variation_mois : number | null;

    constructor(init?: Partial<ComparaisonDynamiqueDTO>) {
    Object.assign(this, init);
    }

}